machine-learning/package.json

```diff
@@ -9,7 +9,7 @@
     "compile": "cp ../tsconfig.json.template ./tsconfig.json && tsc"
   },
   "dependencies": {
-    "firebase-admin": "^9.1.0",
+    "firebase-admin": "git+https://github.com/firebase/firebase-admin-node.git#if-automl-3",
```

Comment:
I know it's temporary, but shouldn't it be #ml-automl ?

Reply:
Oh, my mistake. But that branch worked well enough for me to test my changes.